doc/schema.md

```diff
@@ -17,7 +17,7 @@ The version of the database schema. Schema versioning is split into three stages
 
 ### `pool_hash`
 
-A table for every unique pool key hash. The `id` field of this table is used as foreign keys in other tables.
+A table for every unique pool key hash. The `id` field of this table is used as foreign keys in other tables. The existance of an entry doesn't mean the pool is registered or in fact that is was ever registered.
```

Comment:
Why are you editing this file? Its auto generated from `doc/Schema.hs` . All edits should be made there and then this file generated using `cabal run -- gen-schema-docs >> doc/schema.md`